usagov/usagov-2021
# usagov-1272-leftnav-hover-color: update code for nav hover color

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1272

## Description

Currently the hover color of the left navigation is #F2F3F3. We are consolidating the number of grays that we use on the site. As part of this effort we are changing this hover color gray to #F0F0F0.
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Goto any downlevel page with a leftnav like http://localhost/disaster-housing-shelter
- Verify the hover color for the leftnav is #f0f0f0


## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
